platforms/android/library/src/main/java/io/element/android/wysiwyg/view/StyleConfig.kt

```diff
@@ -68,7 +68,7 @@ data class CodeBlockStyleConfig(
     @Px val leadingMargin: Int,
     @Px val verticalPadding: Int,
     val relativeTextSize: Float,
-    val backgroundDrawable: Drawable,
+    @DrawableRes val backgroundDrawable: Int,
```

Comment:
Switch to storing the resource IDs here so that the style config classes can be compared for equality.